design/general_refactorings.md

```diff
@@ -0,0 +1,381 @@
+## CLI Refactor
+
+
+The purpose of this document is to outline the parts of the Python CLI/SDK code that we would like to refactor as well as provide a design for each point. Through this refactor, we hope that the code will be more readable, and easier to contribute to and maintain.
```

Comment:
for each item

"readable and easier to contribute to and maintain" (without the ,)

Reply:
Fixed